GetMLBAthleteData.py

```diff
@@ -12,8 +12,8 @@
 HEADER = {'Ocp-Apim-Subscription-Key': apiKey }
 SDIO_URL = 'https://api.sportsdata.io/v3/mlb/stats/json/PlayerSeasonStats/2022'
 
-# HOST = 'localhost'
-HOST = '139.99.74.201'
+HOST = 'localhost'
```

Comment:
Revert so this will work in production

Reply:
`db.athletex.io`

Reply:
completed

Reply:
realized we actually only support 443,80 ports to that load balancer